FBSnapshotsViewer/Managers/SnapshotTestResultAcceptor.swift

```diff
@@ -34,7 +34,7 @@ class SnapshotTestResultAcceptor {
         let failedImageSizeSuffix = imagePath.substring(with: failedImageSizeSuffixRange)
         let recordedImageURLs = testResult.build.fbReferenceImageDirectoryURLs.flatMap { fbReferenceImageDirectoryURL -> URL? in
             let url = fbReferenceImageDirectoryURL.appendingPathComponent(testResult.testClassName).appendingPathComponent("\(testResult.testName)\(failedImageSizeSuffix)").appendingPathExtension("png")
-            return fileManager.fileExists(atPath: url.path) ? url : nil
```

Comment:
Do we need to change it?

Reply:
Hmm. This change was the shortest path to implement the desired functionality here, which was to still create test images even if a reference image didn't exist. However, now that I look at it, it seems like maybe it would be better to still throw SnapshotTestResultAcceptorError.notExistedRecordedImage but then catch it and handle it by doing what we want with the autoRecording functionality. I will take another look at this.

Reply:
Thanks @babbage ;)